src/locale/locales/de/messages.po

```diff
@@ -821,7 +783,7 @@ msgstr "Bestätigungscode"
 
 #: src/view/com/modals/Waitlist.tsx:120
 msgid "Confirms signing up {email} to the waitlist"
-msgstr ""
+msgstr "Bestätigt die Eintragung in die Warteliste {email}"
```

Comment:
```suggestion
msgstr "Bestätigt die Eintragung von {email} in die Warteliste"
```

email wasn't a part of the sentence before. it was just randomly added to the end 😂